src/batch/Cargo.toml

```diff
@@ -30,9 +31,11 @@ hytra = "0.1.2"
 icelake = { workspace = true }
 itertools = { workspace = true }
 memcomparable = "0.2"
+opendal = "0.45.1"
```

Comment:
Shall we consider reusing the `risingwave_object_store` crate? Some functionality can be reused, such as monitoring and madsim test. The functionality (read and write object) used in this PR should have been supported in the crate.

Reply:
Currently, I intend to make the batch crate more self-contained and not rely on `risingwave_object_store`, as the way we currently use the underlying opendal is simple and the configuration is oriented towards local disk. In terms of monitoring, I believe we should maintain a separate monitor panel for local disk spilling.